poc-with-identification-counter/api/src/main/java/com/devonfw/application/pocwithidentificationcounter/employeemanagement/service/api/rest/EmployeemanagementRestService.java

```diff
@@ -30,9 +30,9 @@
 	 * @param id the ID of the {@link EmployeeEto}
 	 * @return the {@link EmployeeEto}
 	 */
-	@GET
-	@Path("/employee/{id}/")
-	public EmployeeEto getEmployee(@PathParam("id") CompositeEmployeeKey id);
+	  @GET
+	  @Path("/employee/{employeeId}/company/{companyId}")
+	  public EmployeeEto getEmployee(@PathParam("employeeId") String employeeId, @PathParam("companyId") String companyId);
```

Comment:
It seems you have been facing some technical issues - maybe with JSON mapping or bean-mapping.
These can all be solved properly and easily. See e.g.:
https://github.com/devonfw/devon4j/blob/develop/documentation/guide-json.asciidoc#custom-mapping
If you have issues get it touch. We can even do a skype or teams call to get it resolved.

However, the impact should not change the design and expose the technical problems by chaning the entire API and give up our REST conventions. The idea for the composite keys is IMHO to provide a solution with minimal impact on design, CobiGen templates, etc. Therefore this approach is not leading to the right direction.

Reply:
reverted the changes, put the composite id back to accept the composite id.

Reply:
Replaced the pom files with latest devon (3.2.1) pom files to support new java version (No JAXB on  java 11).